examples/instance_default/versions.tf

```
@@ -0,0 +1,4 @@
+
+terraform {
+  required_version = ">= 0.12"
```

```suggestion
  required_version = ">= 0.12, < 0.13"
```
terraform config should prevent update greater than 0.12 as this module is not tested/known working with versions greater that 0.12

this suggestion do not apply to version constraints inside module: their we want to specify only minimum version.